src/pharmpy/model/external/nonmem/update.py

```diff
@@ -761,7 +777,8 @@ def pk_param_conversion(model: Model, advan, trans):
     d = {}
     for old, new in remap.items():
         d[sympy.Symbol(f'S{old}')] = sympy.Symbol(f'S{new}')
-        d[sympy.Symbol(f'F{old}')] = sympy.Symbol(f'F{new}')
```

Comment:
This is commented out since the bioavailability needs to be moved along with the dose compartment (just as for lag time). If not, then for instance a bioavailability F1 that is moved when adding transit compartment(s) will be renumbered as previous compartment 1 has now become another number (for instance 3 if adding 2 transit comps). This is avoided by instead moving bioavailability with the dose(s)